src/main/java/Duke.java

```diff
@@ -1,10 +1,58 @@
+import java.io.FileNotFoundException;
```

Comment:
Shouldn't the import statements be arranged in lexicographic order?

Reply:
Thank you.